browser/scripts/application.js

```diff
@@ -2178,6 +2198,16 @@ Application.prototype.setupChat = function() {
 
 	this.chatStore = new E2.ChatStore()
 	this.chat = new E2.Chat($('#chat'))
+	$('.chat-users').draggable({
```

Comment:
we actually already have a draggable: https://github.com/vizorvr/vizor-create/blob/develop/browser/scripts/draggable.js